<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Com_Serial</class>
 <widget class="QWidget" name="Com_Serial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Com_Serial</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox_info">
     <property name="title">
      <string>flechazo</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>120</height>
      </size>
     </property>
     <property name="title">
      <string>串口通信</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_config">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>120</height>
         </size>
        </property>
        <property name="title">
         <string>配置</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_config_port">
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>串口：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_config_port">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>280</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_config_baud">
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>波特率：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_config_baud">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_config_data">
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>数据位：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_config_data">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_config_verify">
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>校验位：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_config_veriy">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_config_stop">
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>停止位：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_config_stop">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_operate">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>90</height>
            </size>
           </property>
           <property name="text">
            <string>打开串口</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_com">
        <property name="title">
         <string>通信</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox_com_view">
           <property name="minimumSize">
            <size>
             <width>600</width>
             <height>0</height>
            </size>
           </property>
           <property name="title">
            <string>数据显示</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QTextBrowser" name="textBrowser_com_view"/>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_com_view_config">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>90</height>
               </size>
              </property>
              <property name="title">
               <string>数据操作</string>
              </property>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="3">
                <widget class="QCheckBox" name="checkBox_com_view_config_hexsend">
                 <property name="maximumSize">
                  <size>
                   <width>90</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>16进制发送</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="8">
                <widget class="QPushButton" name="pushButton_com_view_config_send">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>发送</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="5">
                <widget class="QPushButton" name="pushButton_com_view_config_readlog">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>读取</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QLCDNumber" name="lcdNumber_com_view_config_recvnum">
                 <property name="maximumSize">
                  <size>
                   <width>90</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="2" column="2" colspan="4">
                <widget class="QLineEdit" name="lineEdit_com_view_config_senddata">
                 <property name="minimumSize">
                  <size>
                   <width>180</width>
                   <height>30</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="0" column="0" rowspan="2">
                <widget class="QCheckBox" name="checkBox_com_view_config_hexrecv">
                 <property name="maximumSize">
                  <size>
                   <width>90</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>16进制显示</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="8">
                <widget class="QPushButton" name="pushButton_com_view_config_savelog">
                 <property name="minimumSize">
                  <size>
                   <width>90</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>保存</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="4">
                <widget class="QLCDNumber" name="lcdNumber_com_view_config_sendnum">
                 <property name="maximumSize">
                  <size>
                   <width>90</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QCheckBox" name="checkBox_com_view_config_send">
                 <property name="text">
                  <string>发送数据</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QTabWidget" name="tabWidget_com_func">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>Tab 1</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>Tab 2</string>
            </attribute>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
